experiment/posttest.json

```diff
@@ -0,0 +1,18 @@
+[
+{"question":"The final addition sum of the numbers, 0110 & 0110 is ?",
+"answers":{
+            "a":"1101",
+            "b":"1001",
+            "c":"1111",
+            "d":"1010"
+          },
+"correctAnswer":"a"},
```

Comment:
Incorrect answer to the question, the correct answer should be 1100, which is not one of the options.